test/SCRStub.rb

```diff
@@ -58,3 +58,25 @@ def stub_scr_read(path_name)
     allow(Yast::SCR).to receive(:Read).with(path).and_return info
   end
 end
+
+module YaPINetworkStub
```

Comment:
still unchanged

btw stubbing of NetworkInterfaces is so huge that it makes sense to to replace whole class by a nil object